### PR TITLE
Adding support for decimeters

### PR DESCRIPTION
**Pull request by arutrub**
Please let me know if there are additional items I need to complete before this is approved. 
All tests pass

### CODE REVIEW

File: test/best.js [hunk 1]
```diff
@@ -20,10 +20,10 @@ test('excludes measurements', (t) => {
   t.plan(1)
   const actual = convert(1200000).from('mm').toBest({ exclude: ['km', 'm'] })
   const expected = {
-    val: 120000,
-    unit: 'cm',
-    singular: 'Centimeter',
-    plural: 'Centimeters'
+    val: 12000,
```
Comment (Eomm) [inline test/best.js]: Could you update the `exclude` parameter only?

File: test/best.js [hunk 2]
```diff
@@ -59,10 +59,10 @@ test('pre-cut off number', (t) => {
   t.plan(1)
   const actual = convert(9000).from('mm').toBest({ cutOffNumber: 10 })
   const expected = {
-    val: 900,
-    unit: 'cm',
-    singular: 'Centimeter',
-    plural: 'Centimeters'
+    val: 90,
```
Comment (Eomm) [inline test/best.js]: Same here: could you update just the `cutOffNumber` to `100` to get the same expected response?

**Review by Eomm — COMMENTED** [review]
I would just leave some changed tests with the old `expected` value, then this PR would be perfect!

Thanks!

**Comment by arutrub** [comment]
Thank you for the suggestions! 
I have implemented the requested changes, and I have updated the README to reflect them as well. I left in one for converting to decimeters, if you want me to revert that change, I certainly can! 🙂

**Review by Eomm — APPROVED** [review]
LGTM

**Pull request merged**